aten/src/ATen/core/Tensor.h

```diff
@@ -183,7 +183,7 @@ class CAFFE2_API Tensor {
   }
 
   at::MemoryFormat suggest_memory_format() const {
```

Comment:
So, what are the new semantics of `suggest_memory_format`? Can we get a docstring explaining what exactly this function does, and when you should call it?